data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,277 @@ mission "Arachi Cartoonist 2"
+			label notime
+			`	She mumbles something that the box doesn't pick up, and then says, "Cost me the match, impatience did," she says. "Still, much younger than most Kimek there, I am. Glad to have gotten so far, I feel."`
+				goto coop-game
+			
+			label prettygood
+			`	"Thank you. Glad to have gotten so far, I feel. Disappointing, defeat is, but try harder next year, I shall," she says."`
+				goto coop-game
+			
+			label why
+			`	"For the Kimek victors only, the cooperative match is reserved. Team up with Tumub and Ulmodi, my opponent will."`
+			
+			label coop-game
+			`	Once the mechanics are done, the ring gradually rises up in the air, and then fully takes off, leaving the hangar. Out in the open, you're overlooking a massive track separated from the surrounding wilderness by a perfectly rectangular border of trees. Immediately under you are the contestants, in twenty groups of three, close to a starting line. As you're brought closer to the teams, small monitors light up in front of you, and Pairri inputs the command for yours to focus on Tumub and Ulmodi, who are in a team with the Kimek that beat Pairri in her match.`
+			`	The monitor also allows you to switch to a camera view of the entire stage: a long, mostly flat stretch of land, leading to a rocky wall that is cut to form a perfect right angle with the ground. At the top, there is a river bank with many logs laid on the ground. The river itself is not terribly big, but it leads to a waterfall, and the current is strong enough that the many boats in the waters need to constantly adjust their position to avoid hitting the various rocks that protrude far out of the water.`
+			`	Pairri draws your attention back to the line, where judges have the twenty teams line up. A chariot carrying the other two members of each team is attached to the Saryds, who seem to be working themselves up, hitting their hoofs repeatedly on the ground. Meanwhile, the Kimek and Arachi are securing large bags and pieces of fabric to their chariots.`
+			`	"In many ways we differ, when it comes to physical form," Pairri says. "So in the cooperative match, different tasks, each member of a team is assigned. For the sake of fairness, it is so."`
+			choice
+				`	"What are they carrying in those bags?"`
+				`	"So is pulling those chariots the 'task' of the Saryds?"`
+			`	A loud airhorn interrupts Pairri as she is about to answer, and the Saryds start galloping toward the wall. The Kimek and Arachi of each chariot do their best to stay on, as their Saryd teammates press on with little regard for the chariot's stability, focused on reaching the wall as fast as possible.`
+			`	It's a very tight race, but the Saryds begin breaking once they're getting closer, out of both exhaustion, and a need to avoid crashing into the wall. Once they arrive at the base, they immediately lay down to rest as the Arachi of each team unstrap the chariots, and the Kimek begin climbing the wall, carrying their small bags.`
```

Comment:
I think one actually crashing into the wall would be a bit too much for an otherwise really tame mission.

Reply:
Easy enough to nix or even adjust to just a less serious collision. Let me know which you prefer. :)

Really just added it as a tie back to the one swiping the tree since the need for braking was already emphasized there in the first version.

Reply:
I'd just go without it, this portion is a race in an open field, and the athletes know how the games go, so this should be one of the easiest portions to train for, as it is only a sprint (granted a very taxing one) where they need to slow down at a point. Given only 20 Saryds make it to this portion, that'd also reduce the likelihood of any of them messing this part up, given they were the best of their own game.

Reply:
Done.